package.json

```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```

Comment:
@harshit98 yes I know, This happened in my local version too, I chose not to include it in my PRs. That is why I wasn't sure if this should have been excluded or not, maybe we should ask someone else.